stable/jenkins/templates/jenkins-master-deployment.yaml

```diff
@@ -61,6 +61,9 @@ spec:
       {{- end }}
       securityContext:
         runAsUser: 0
+      {{- if .Values.rbac.install }}
+      serviceAccountName: {{ template "fullname" . }}
+      {{- end }}
```

Comment:
> looks like in the other PR the value was rbac.create vs rbac.install here. Should I make that change as well?

I like that idea (consistency ftw), but let's ping @lachie83. This chart is his baby.